src/BannedSymbols.txt

```diff
@@ -9,3 +9,8 @@ M:Microsoft.VisualStudio.ProjectSystem.CommonProjectSystemTools.LoadedProject(Mi
 
 T:Microsoft.VisualStudio.ProjectSystem.ProjectAutoLoadAttribute; Using IProjectDynamicLoadComponent avoids accidently reloading the project while loading, switching branches, installing NuGet package, etc.
 T:Microsoft.VisualStudio.ProjectSystem.ConfiguredProjectAutoLoadAttribute; Using IProjectDynamicLoadComponent avoids accidently reloading the project while loading, switching branches, installing NuGet package, etc.
+
+M:System.Threading.Tasks.Task.ConfigureAwait(System.Boolean); "ConfigureAwait(true)" should be removed, and "ConfigureAwait(false)" should be replaced with "await TaskScheduler.Default"
```

Comment:
This is most likely a nit, but ConfigureAwait(false) should be _preceeded_ by a call to `await TaskScheduler.Default`. 

ie 

```
await SomeThing().ConfigureAwait(false);
```
should become
```
await TaskScheduler.Default;
await SomeThing();
```

Reply:
Hmm, yeah, I don't think we can fit the explanation of how to do this properly. I'll back out the last commit. If someone knows how to fit that into the one-liner, we can improve it.